### DISCUSSION TITLE
Error: Please install the package(s) for your selected database type: "mysql2"

### DISCUSSION THREAD

**Discussion by Bazimek** [discord-tickets/bot#214] [Support (Q&A)] [answered]
How do I fix this? Installed it to the pterodactyl egg.

**Comment by vincent-cahart** [comment]
same problem for me

**Comment by eartharoid** [comment, marked as answer]
If you're using the Discord Tickets egg for Pterodactyl, add `mysql2` to the "Plugins" field on the server's startup page. If you're using a generic Node.js or discord.js egg, add it to the field that says additional npm/node packages.
